plugins/main/public/controllers/overview/index.js

```diff
@@ -28,7 +27,6 @@ SelectAgent.displayName = 'SelectAgent';
 RequirementCard.displayName = 'RequirementCard';
 
 app
-  .controller('overviewController', OverviewController)
   .value('OverviewWelcome', OverviewWelcome)
```

Comment:
**nitpick**: this value seems that is not used anymore, we could remove it.

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6594/commits/f22e354f610475555fb056b08ea3b5ceca715c91

Reply:
Regarding the commit, I don't see the `OverviewWelcome` value was removed.